mongoose/groups/GroupSchema.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * @file Implements mongoose schema for groups.
+ */
+ import mongoose, {Schema} from "mongoose";
+ import Group from "../../models/group/Group"
+ 
+ const GroupSchema = new mongoose.Schema<Group>({
+     members: {type:[]},
```

Comment:
For lines 8 and 10 do we need to explicitly define the array type?

Reply:
i tried doing something like the below for the array of strings, but im getting an error. I'm thinking of leaving it generic for now to see if it updates correctly in mongo when we test

https://stackoverflow.com/questions/45780272/how-to-define-an-array-of-strings-in-typescript-interface

Reply:
Fair enough so yeah let's just keep it in mind as we progress to the next stages.
If we run into issues then we'll come back to check this

Reply:
(unresolved it so others can check/give their feedback)